utils/build-presets.ini

```diff
@@ -1273,8 +1273,6 @@ install-llbuild
 install-swiftpm
 install-swift-driver
 install-swiftsyntax
-install-skstresstester
```

Comment:
Should we also not build it if we are not installing it? 
```
skstresstester
swiftevolve
```

Reply:
Oh, yes that makes sense. I was thinking that we should still build the projects in CI to make sure they don’t break but of course that’s already covered by other presets.